web/src/components/molecules/Content/Table/index.tsx

```diff
@@ -92,6 +96,8 @@ const ContentTable: React.FC<Props> = ({
   pageSize,
   requests,
   addItemToRequestModalShown,
+  columns,
+  setColumns,
```

Comment:
Instead of passing these as props you can move the useState from the hooks to this file since it is only used here.

Reply:
It's because it will be used for integrating into views menu

Reply:
OK, so you can keep it the way it is.